docs/api/stories.md

```diff
@@ -0,0 +1,497 @@
+---
+title: 'Stories'
+---
+
+### Component Story Format (CSF)
+
+Storybook's Component Story Format (CSF) is the recommended way to [write stories](../writing-stories/introduction) since Storybook 5.2. [Read the announcement](https://medium.com/storybookjs/component-story-format-66f4c32366df) to learn more about how it came to be.
+
+<div style="background-color:#F8FAFC">
+TODO: add advanced readme link and if stories of api needs to be maintained)
```

Comment:
We probably need to create this advanced README and put the old docs content in it. I think it should probably be in `lib/core/ADVANCED-README.md`

Reply:
Added the link, let me know if it's ok.